### PR TITLE
Package allthekernels

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/allthekernels/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  description: |
+    Specify which kernel a cell should use with >kernelname. If no kernel is specified, IPython will be used.
+  doc_url: https://github.com/minrk/allthekernels
+  dev_url: https://github.com/minrk/allthekernels
+
+extra:
+  recipe-maintainers:
+    - SylvainCorlay
+    - wolfv
+    - minrk
```
[the remark this answers is not in the document]
Comment (SylvainCorlay) [reply recipes/allthekernels/meta.yaml]: I just got confirmation from minrk on gitter.

[the remark this answers is not in the document]
Comment (wolfv) [reply recipes/allthekernels/meta.yaml]: Yep!

**Comment by SylvainCorlay** [comment]
Please hold on to merge. I am double checking something.

**Comment by marcelotrevisani** [comment]
> Please hold on to merge. I am double checking something.

Please, let us know when it is ready! :)

File: recipes/allthekernels/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (SylvainCorlay) [reply recipes/allthekernels/meta.yaml]: > do you have a link?

no, but you may notice the thumbs up by minrk on your comment above :)

**Pull request merged**